### PR TITLE
Add apollo custom extension to replace `apollo_` prefix

### PR DESCRIPTION
**Pull request by yuzushioh**
Added `ApolloExtension` and replace `apollo_` prefix by `apollo.` ref: #1170

### CODE REVIEW

**Comment by yuzushioh** [comment]
@designatednerd `testWatchedQueryGetsUpdatedWithListReorderingFromOtherQuery ` is failing for some reason. I am not sure if it's because of my change

**Comment by designatednerd** [comment]
@yuzushioh Prooooobably not, that test is super flaky, I'll kick it

File: Sources/ApolloCodegenLib/OptionalBoolean.swift [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import Foundation
 
 extension Optional where Wrapped == Bool {
-  
-  var apollo_boolValue: Bool {
+  var valueOrFalseIfNone: Bool {
```
Comment (designatednerd) [inline Sources/ApolloCodegenLib/OptionalBoolean.swift:4]: I'd still like to keep this as `boolValue` - ultimately it behaves similarly to `[NSNumber boolValue]` in that if you call it on `nil` it returns false.

Comment (yuzushioh) [reply Sources/ApolloCodegenLib/OptionalBoolean.swift]: OK. I changed it to `boolValue` with a comment on it.

**Pull request merged**